app/(pages)/marketplace/page.tsx

```diff
@@ -69,33 +98,61 @@ const MarketplacePage = () => {
         });
         return Assetdata;
       };
+
+      const OnChainPointsCalculations = async () => {
```

Comment:
please update the function names to start with lowercase

Reply:
only if the function is a Functional Component, then it should start with capital character